connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskThreadedTest.java

```diff
@@ -147,7 +150,6 @@ public void testPollsInBackground() throws Exception {
             workerTask.iteration();
         }
         workerTask.stop();
-        workerTask.awaitStop(Long.MAX_VALUE);
```

Comment:
Yeah, I realized that too. We can probably merge the tests in `WorkerSinkTaskTest` now (if there's no need for test cases which actually involve a background thread). If it's OK, maybe we could address this in a follow-up?